### DISCUSSION TITLE
[dumb-init] usrsrcappEntrystart.sh No such file or directory [May 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2256] [Discord Threads]
> 🙋‍♂️ **jiub**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### lemme DM u  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979342632901804062)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by ryantando** [comment]
@dstala have you found a solution on this?
